doc/classes/OS.xml

```diff
@@ -265,7 +265,8 @@
 			<return type="String">
 			</return>
 			<description>
-				Returns the host OS locale.
+				Returns the host OS locale as a string of the form [code]locale[_VARIANT][/code] (e.g. [code]en_US[/code] for American English). Note that the variant may not always be returned depending on the OS and user configuration. See [url=https://docs.godotengine.org/en/latest/tutorials/i18n/locales.html]this page[/url] for examples of locale codes that can be returned.
```

Comment:
In some cases locale can have `lang-Script-COUNTRY` format (e.g `zh-Hans-CN`, `az-Cyrl-AZ`), at least on macOS.

And Linux `LANG` variable can have additional currency info after `@` (e.g `en_IE@euro`).